### PR TITLE
Added video for PAT

### PR DESCRIPTION
**Pull request by jerae-duffin**
Added PAT video

### CODE REVIEW

File: docker-hub/access-tokens.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ any time.
 
 ## Create an access token
 
+The following video walks you through the process of Managing Access Tokens.
```
Comment (thaJeztah) [inline docker-hub/access-tokens.md:40]: Not sure if these need to start with a capital;

```suggestion
The following video walks you through the process of managing access tokens.
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]
left one suggestion, but otherwise LGTM

**Review by usha-mandya — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**